core/src/main/java/org/wso2/msf4j/util/Utils.java

```diff
@@ -114,4 +114,38 @@ public static int getIntersection(Set set1, Set set2) {
         cloneSet2.retainAll(cloneSet1);
         return cloneSet2.size();
     }
+
+    /**
+     * Get the path without leading or trailing / and with multiple / reduced to one /
+     *
+     * @param path The path
```

Comment:
Would it be clear if we say, `resource path`?

Reply:
You want to just change the comment or also the method name?